HW0/P6/P6.txt

```diff
@@ -1 +1,8 @@
-# Your answers here
+P6A
```

Comment:
You should not be seeing speed-ups of 16x when you are using 4 processes.  This should cap at 4x